src/transformers/models/prophetnet/modeling_prophetnet.py

```diff
@@ -160,12 +161,14 @@ def ngram_attention_bias(sequence_length, ngram, device, dtype):
     """
     This function computes the bias for the predict stream
     """
-    bias = torch.ones((ngram, sequence_length, 2 * sequence_length), device=device, dtype=dtype) * float("-inf")
+    bias_array = np.ones((ngram, sequence_length, 2 * sequence_length), dtype=np.float) * np.float("-inf")
```

Comment:
Here we go from pytorch tensor operations to numpy => does this work on GPU as expected? And is the slow-down insignificant? @mfuntowicz it would be great if you can take a look here as well

Reply:
This tensor seems constant to me, I would compute this one at init time and store it using a buffer to avoid the cost of creation.

Reply:
We are now improving pytorch-onnx converter by [PR](https://github.com/pytorch/pytorch/pull/48734), and will convert this model through scripting. Then we don't need convert to numpy anymore. Currently I put the title to [WIP]. Will let you know if we need further review. Thanks for your time.